src/css/style.css

```diff
@@ -0,0 +1,368 @@
+  align-items: inherit;
+  list-style: none;
+}
+
+li {
+  list-style: none;
+}
+
+nav ul li {
+  margin-right: 40px;
+}
+
+nav ul li a {
+  background: #fff;
+  padding: 8px;
+  border-radius: 6px;
+  text-decoration: none;
+  color: black;
+}
+
+nav a:hover {
+  transition: 0.3s;
+  background: black;
+  color: #fff;
+}
+
+.logo {
+  width: 100px;
+  height: 40px;
+  border-radius: 50px;
+  background: url('../asset/logo.png') center center/cover no-repeat;
+}
+
+.main-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  top: 60px;
+  margin: 20px 15px 200px 5px;
+}
+
+.main-container-sup {
+  width: 270px;
+  border: 5px solid rgb(145, 93, 93);
+  margin-left: 10px;
+  margin-bottom: 20px;
+  border-radius: 15px;
+}
+
+.new-comment-form button {
+  width: 70%;
+  align-self: center;
+  cursor: pointer;
+}
+
+.movie-banner {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.like-movies {
+  width: 40px;
+  margin-right: 10px;
+  transition: 1s;
+  margin-bottom: 10px;
+}
+
+.like-movies i {
+  font-size: 24px;
+  margin-top: 10px;
+}
+
+.modal-image {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 500px;
+  margin: 70px auto 40px auto;
+}
+
+.movie-banner img {
+  width: 250px;
+  height: 300px;
+  border: 3px solid rgb(121, 94, 94);
+  border-radius: 15px;
+}
+
+.modal-image img {
+  width: 300px;
+  height: 300px;
+  border: 1px solid black;
+  border-radius: 10px;
+}
+
+.movie-banner img:hover {
+  border: 5px solid rgb(227, 30, 30);
+}
+
+.details-info {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.details-info h2 {
+  font-size: 24px;
+  font-family: 'Times New Roman', Times, serif;
+  font-weight: 700;
+  margin: 10px 10px;
+}
+
+.like-doc {
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 20px;
+  margin-top: -10px;
+  margin-bottom: 5px;
+  font-size: 16px;
+  color: rgb(255, 42, 0);
+}
+
+.fa-solid {
+  color: red;
+}
+
+.comment-rese {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 10px 20px;
+}
+
+.comment-rese button {
+  background: #fff;
+  border: 1px solid blue;
+  color: blue;
+  padding: 8px 12px;
+  font-size: 16px;
+  border-radius: 12px;
+  transition: 0.5s;
+}
+
+.add-comment button {
+  background: #fff;
+  padding: 8px 12px;
+  border: 1px solid blue;
+  width: auto;
+  margin: 5px 0 40px 0;
+  font-size: 18px;
+  border-radius: 15px;
+}
+
+.comment-rese button:hover {
+  background: blue;
+  color: #fff;
+}
+
+footer {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: rgb(40, 38, 38);
+}
+
+.footer p {
+  transition: 1s;
+  font-family: 'Times New Roman', Times, serif;
+  font-size: 16px;
+  color: #fff;
+}
+
+.footer p:hover {
+  font-size: 24px;
+}
```

Comment:
- [ ] OPTIONAL
Going by the mockup given, the content of your footer should be aligned to the left of the page.

![image](https://user-images.githubusercontent.com/12745474/205323692-2256bc8d-203e-423e-9dba-51d4bc5cc83c.png)